developer-tooling/scripts/optimize_runpython.py

```
@@ -0,0 +1,239 @@
+                                        continue
+
+                                    # Rewrite AddFieldOrSkip to migrations.AddField
+                                    if isinstance(op, ast.Call) and isinstance(op.func, ast.Name) and op.func.id == "AddFieldOrSkip":
+                                        op.func = ast.Attribute(
+                                            value=ast.Name(id="migrations", ctx=ast.Load()),
+                                            attr="AddField",
+                                            ctx=ast.Load()
+                                        )
+                                    new_ops.append(op)
+                                class_node.value.elts = new_ops
+                                print(f"  -> Part block 2 operations optimized. Removed RemoveFieldOrSkip and converted AddFieldOrSkip to AddField.")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)